tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -15,39 +17,55 @@
                 <ResourceDictionary Source="ms-appx:///DevHome.Common/Environments/Templates/EnvironmentsTemplates.xaml" />
             </ResourceDictionary.MergedDictionaries>
 
-            <converters:BoolNegationConverter x:Key="BoolNegationConverter" />
             <converters:BoolToVisibilityConverter x:Key="CollapsedWhenTrueBoolToVisibilityConverter" TrueValue="Collapsed" FalseValue="Visible"/>
-            <converters:EmptyCollectionToObjectConverter x:Key="NegatedEmptyCollectionVisibilityConverter" EmptyValue="Visible" NotEmptyValue="Collapsed"/>
-            <converters:EmptyCollectionToObjectConverter x:Key="EmptyCollectionVisibilityConverter" EmptyValue="Collapsed" NotEmptyValue="Visible"/>
-            <converters:EmptyObjectToObjectConverter x:Key="EmptyObjectToObjectConverter" NotEmptyValue="Visible" EmptyValue="Collapsed"/>
-            <converters:EmptyStringToObjectConverter x:Key="EmptyStringToObjectConverter" NotEmptyValue="Visible" EmptyValue="Collapsed"/>
 
             <!--- Template for dev drive horizontal cards on the dev insights page. -->
             <DataTemplate x:Key="HorizontalCardForDevDriveInsightsPage" x:DataType="devViewModels:DevDriveCardViewModel">
-                <ctControls:SettingsCard Header="{Binding DevDriveLabel, Mode=OneWay}">
-                    <ctControls:SettingsCard.HeaderIcon>
+                <ctControls:SettingsExpander IsExpanded="False" Header="{Binding DevDriveLabel, Mode=OneWay}" >
```

Comment:
@krschau the expander is needed when the dev drive is untrusted, see figma here- [https://www.figma.com/file/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?type=design&node-id=3851-174372&mode=design&t=gjIoBJpEHj0T4zzq-4](https://www.figma.com/file/AZ03I5bFPmrOpjVzFRjVn4/Dev-Home-Settings?type=design&node-id=3851-174372&mode=design&t=gjIoBJpEHj0T4zzq-4)
However when the drive is trusted, we do have anything to show in the expander. A bug was filed and fixed for this issue earlier. 

Do you know if we can change to a SettingsCard instead of the expander dynamically in this situation?

Reply:
I think the way to handle this is to have a SettingsExpander *and* a SettingsCard, and only show one at a time. You can bind their visibility to IsDevDriveTrusted.

Reply:
Sure, that would work. Will update.